test-framework/remote/pom.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<!--
+~ Copyright 2016 Red Hat, Inc. and/or its affiliates
+~ and other contributors as indicated by the @author tags.
+~
+~ Licensed under the Apache License, Version 2.0 (the "License");
+~ you may not use this file except in compliance with the License.
+~ You may obtain a copy of the License at
+~
+~ http://www.apache.org/licenses/LICENSE-2.0
+~
+~ Unless required by applicable law or agreed to in writing, software
+~ distributed under the License is distributed on an "AS IS" BASIS,
+~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~ See the License for the specific language governing permissions and
+~ limitations under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>keycloak-test-framework-parent</artifactId>
+        <groupId>org.keycloak.test</groupId>
+        <version>999.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>keycloak-test-framework-remote</artifactId>
+    <name>Keycloak Test Framework - Remote testing utils</name>
+    <packaging>jar</packaging>
+    <description>Utility Keycloak server Suppliers for the Keycloak Test Framework</description>
+
+    <dependencies>
+        <dependency>
```

Comment:
You need to add a dependency here on the remote-providers, as that will in turn make any test module that depends on this to have a dependency on the remote-providers. That's probably the last piece to the puzzle to get this one to pass on GA

Reply:
Yes, I resolved the pom dependency locally. It works running with maven (`mvn install` and `mvn test`)